.github/workflows/gcp_image.yml

```
@@ -31,7 +52,11 @@ jobs:
         run: |
           TIMESTAMP="$(date -u +'%Y%m%d%H%M%S')"
           echo "TIMESTAMP=$TIMESTAMP" >> "$GITHUB_OUTPUT"
+          if [ "${{ inputs.techno }}" == "sev" ]; then
           echo "IMAGE_NAME=alpha-$TIMESTAMP-cosmian-vm-sev-${{ inputs.distrib }}" >> "$GITHUB_OUTPUT"
+          elif [ "${{ inputs.techno }}" == "tdx" ]; then
+          echo "IMAGE_NAME=alpha-$TIMESTAMP-cosmian-vm-tdx-${{ inputs.distrib }}" >> "$GITHUB_OUTPUT"
+          fi
```

could we merge this in 1 block? (by using ${{ inputs.techno }})?
 ```suggestion
          echo "IMAGE_NAME=alpha-$TIMESTAMP-cosmian-vm-${{ inputs.techno }}-${{ inputs.distrib }}" >> "$GITHUB_OUTPUT"
```